api.js

```diff
@@ -13,8 +13,10 @@ var commonPathPrefix = require('common-path-prefix');
 var resolveCwd = require('resolve-cwd');
 var uniqueTempDir = require('unique-temp-dir');
 var findCacheDir = require('find-cache-dir');
+var debounce = require('lodash.debounce');
```

Comment:
`lodash.debounce` contains only `debounce()` function, while `lodash` module contains all of them.